index.js

```diff
@@ -0,0 +1,34 @@
+const myModule = require('./myModule.js');
+var 
+gm = require('gm');
+
+var favoriteFoods = myModule.favoriteFoods();
+
+for (let i = 0; i < favoriteFoods.length; i++) {
+        console.log(favoriteFoods[i]);
+    }
+
+// console.log the width and height of the image
+gm('./welcome2.png')
+.size(function (err, size) {
+    if (!err)
+    console.log(size.width);
+    console.log(size.height)
+});
+
+// annotate an image
+gm('./brandNewImg.jpg')
+.stroke("black")
+.drawCircle(10, 10, 20, 10)
+.font("Helvetica.ttf", 12)
+.drawText(30, 20, "Got it!")
+.write("./brandNewImg2.png", function (err) {
+  if (!err) console.log('done');
```

Comment:
This is shortened syntax - because the if statement is entirely on one line, you do not have to include the curly braces for the console.log here.  You COULD choose to include the curly braces here but they are not required.  So this syntax works whereas there is an issue with lines 14-16.